### ISSUE TITLE
The default kamon.datadog.flush-interval is smaller than kamon.metrics.tick-interval

### ISSUE THREAD

**Issue by richardimaoka** [kamon-io/kamon-datadog#12]
`kamon.metric.tick-interval = 60 seconds` [in kamon](https://github.com/kamon-io/Kamon/blob/d9ae0f4d50763017b8d33dfcb6094375a30cb361/kamon-core/src/main/resources/reference.conf#L42), 

but the default `kamon.datadog.flush-interval` is not compliant with the following:

```
# Interval between metrics data flushes to Datadog. It's value must be equal or greater than the
# kamon.metric.tick-interval setting.
```

https://github.com/kamon-io/kamon-datadog/blob/017526496d98e7603884bf221b4f46717cda8844/src/main/resources/reference.conf#L28

**Comment by dominics** [comment]
Looks like this is fixed in 1.0.0 (metric reporters now all use the same schedule of ticks)

**Comment by eugenemiretsky** [comment]
@dominics is that true? Both settings are still exposed in 1.0.1
